client/src/components/ChannelInfo/index.css

```diff
@@ -0,0 +1,57 @@
+.repo-info-wrapper {
+  height: -webkit-fill-available;
+}
+
+.repo-info-header {
+  color: #9a9ea4;
+  font-size: 1.2em;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-weight: bold;
+}
+
+.repo-info-description {
+  color: #9a9ea4;
+  margin-bottom: 20px;
+}
+
+.repo-info-stats {
+  display: flex;
+  justify-content: space-between;
+}
+
+.repo-info-stats a {
+  color: #9a9ea4;
+  text-decoration: none;
+}
+
+.repo-info-stats a:hover {
+  text-decoration: underline;
+}
+
+.private-repo-message {
+  font-size: 0.95em;
+  color: #9a9ea4;
+  line-height: 25px;
+}
+
+.channel-info-tab {
+  width: 50%;
+  color: #70747b !important;
+}
+
+.tabpanel-box {
+  padding: 15px 20px 0px 20px;
+}
+
+.active-tab {
+  color: #9a9ea4 !important;
+}
+
+.channel-info-wrapper {
+  max-height: 50%;
+  overflow-wrap: break-word;
+  overflow-y: scroll;
```

Comment:
Set to `auto` to hide the inactive scrollbar.

![image](https://user-images.githubusercontent.com/51796498/110286214-8edfec00-800a-11eb-8f87-b144851d28b3.png)